improver/wxcode/wxcode_decision_tree.py

```diff
@@ -129,86 +125,88 @@ def wxcode_decision_tree():
             "diagnostic_conditions": ["above"],
         },
         # A.1.1
-        "heavy_sleet_continuous": {
-            "succeed": 18,
-            "fail": "heavy_rain_or_snow_continuous",
-            "probability_thresholds": [0.0, 0.0],
-            "threshold_condition": ">",
-            "condition_combination": "AND",
+        "heavy_snow_continuous": {
+            "succeed": 27,
+            "fail": "heavy_rain_or_sleet_continuous",
+            "probability_thresholds": [0.0],
+            "threshold_condition": "<",
```

Comment:
I'm not sure this change in behaviour is picked up by a unit test (although it is by the acceptance tests). Is that intentional?

Reply:
I suspect that not every pathway through the weather symbols code is tested. I was trying to avoid a wholesale rewrite. How important do you think this is?

Reply:
Yes, I expect you're right. Unless we can find a small tweak to make to one of the points in the existing unit tests, or it's easy to add a simple new test we new data I'm ok to leave it. It is covered by the acceptance test data at the moment, so that's something.